test/new_relic/telemetry_sdk/clients/client_test.rb

```diff
@@ -0,0 +1,71 @@
+# encoding: utf-8
+# frozen_string_literal: true
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-telemetry-sdk-ruby/blob/main/LICENSE for complete details.
+
+require File.expand_path(File.join(File.dirname(__FILE__),'../../..','test_helper'))
+require 'new_relic/telemetry_sdk/clients/client'
+require 'json'
```

Comment:
necessary?  since test_helper requires the main project file, which presumably has all the requires necessary to load all the files from the lib folder.